testing_lang/testing_language.py

```diff
@@ -149,7 +151,7 @@ def withdraw(self, tx_id, oindex, exitor):
 
             self.root_chain.transact({
                 'from': exitor['address']
-            }).startDepositExit(utxo_pos + 1, deposit_amount)
+            }).startDepositExit(utxo_pos + 1, NULL_ADDRESS, deposit_amount)
```

Comment:
Name ETH instead of NULL_ADDRESS is more meaningful.
How about we switch the order of token and amount arguments both in python code and the contract. That way, we could benefit from python default args.